### ISSUE TITLE
ValidateUpdateListItem - suddenly stopped preventing loops

### ISSUE THREAD

**Issue by damooji03** [SharePoint/sp-dev-docs#7527]
### Target SharePoint environment

SharePoint Online

### What SharePoint development model, framework, SDK or API is this about?

SharePoint REST API

### Developer environment

Windows

### What browser(s) / client(s) have you tested

- [ ] 💥 Internet Explorer
- [X] 💥 Microsoft Edge
- [X] 💥 Google Chrome
- [ ] 💥 FireFox
- [ ] 💥 Safari
- [ ] mobile (iOS/iPadOS)
- [ ] mobile (Android)
- [ ] not applicable
- [ ] other (enter in the "Additional environment details" area below)

### Additional environment details

- PowerAutomate

### Describe the bug / error

HTTP request action in flow used to update a SharePoint item using the ValidateUpdateListItem endpoint is causing the flow to loop. My team has many flows configured this way and have been running them for over a year with no issue, however recently (we think around 11/12/21) the flows started looping. 

### Steps to reproduce

1. Create SharePoint list
2. Create Power Automate flow > Automated cloud flow > When item is created or modified

Within the flow:
1. Set the site and list name
2. Add action 'Convert time zone'
3. Add action 'Send an HTTP request to SharePoint'
4. Select the same site address from step 1
5. **Method:** Post
6. **Uri**: _api/web/lists/getByTitle('<insert list name>')/items(@{triggerOutputs()?['body/ID']})/validateupdatelistitem()
7. **Headers**: nothing but some people enter`{
  "Accept": "application/json;odata=verbose",
  "Content-Type": "application/json;odata=verbose"
}`
8. **body**: `{
    "formValues": [
        {
            "FieldName": "Title",
            "FieldValue": "test@{triggerOutputs()?['body/ID']}"
        },
        {
            "FieldName": "Editor",
            "FieldValue": "[{'Key':'@{triggerOutputs()?['body/Editor/Email']}'}]"
        },
        {
            "FieldName": "Modified",
            "FieldValue": "@{body('Convert_time_zone')}"
        }
    ],
    "bNewDocumentUpdate": false
}`

### Expected behavior

The REST call updates the item without triggering the flow. This has been working for over a year for 80+ flows we have, and suddenly we started getting floods of emails from users complaining about issues.

**Comment by influential-eliot** [comment]
Heya, this one is causing issues for two of my clients, just hoping that it is on the table soon.

**Comment by mblennegard** [comment]
Seems like a duplicate to issue #7424 

This is still very much an issue though.

**Comment by influential-eliot** [comment]
Yes, but I think that these issues are unfortunately for problems with the documents, that's the only reason that I mentioned it.

> Seems like a duplicate to issue #7424
> 
> This is still very much an issue though.

(_despite seemingly being the only public resource that we have to log such frustrations_)

**FWIW** ... I think the issue is with whatever they have changed in how Power Automate is subscribing to List updates. Not a SharePoint issue per se.
